mteb/tasks/Classification/multilingual/AfriSentiClassification.py

```diff
@@ -65,7 +65,7 @@ def load_data(self, **kwargs):
         if self.data_loaded:
             return
         self.dataset = {}
-        for lang in self.langs:
+        for lang in self.hf_subsets:
```

Comment:
ahh thanks good catch - I did the PR that changes this - should be more careful in the future.